packages/cleanup-plugin/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@wonderlandengine/publish-plugin",
+    "version": "0.0.1",
+    "description": "Wonderland Editor plugin to publish an application to the Wonderland Cloud",
+    "main": "index.mjs",
+    "typings": "index.d.ts",
+    "type": "module",
+    "module": "index.mjs",
+    "keywords": [
+        "wonderland-engine"
+    ],
+    "scripts": {
+        "tsc": "tsc --noEmit"
```

Comment:
Maybe you should add a `postinstall` script to run this script to ensure it's valid? I know we prebundle with esbuild, but esbuild doesn't typecheck